libr/anal/cc.c

```diff
@@ -100,10 +100,12 @@ R_API bool r_anal_cc_exist(RAnal *anal, const char *convention) {
 }
 
 R_API const char *r_anal_cc_arg(RAnal *anal, const char *convention, int n) {
-	r_return_val_if_fail (anal && convention, NULL);
-	if (n < 0) {
+	r_return_val_if_fail (anal, NULL);
+	r_return_val_if_fail (n >= 0, NULL);
+	if (!convention) {
```

Comment:
if there's no convention it is probably going to be problematic in the analysis later. maybe worth having a warning with eprintf

Reply:
(only once)

Reply:
Agree, it would be handy for catching the problems.

Reply:
There is already a warning. It is not here because otherwise it would be printed multiple times.